### DISCUSSION TITLE
为什么更新后无法提取 v5.2.2 的压缩包？之前的 v5.0.0 都是可以正常提取的

### DISCUSSION THREAD

**Discussion by moegirlwangge** [SlopeCraft/SlopeCraft#112] [Q&A｜问答] [answered]
![屏幕截图 2024-03-25 142633](https://github.com/SlopeCraft/SlopeCraft/assets/69033087/31a5d075-8af1-428b-8c06-78111c2f507e)

**Comment by ToKiNoBug** [comment, marked as answer]
建议换一个解压软件，比如7zip。Windows还不能正确解压zip中用lzma算法压缩的文件，然后报这么个人类不可读的错误

**Reply by moegirlwangge** [reply]
感谢🙏！现在这个问题已经解决了
造成这种问题的原因是因为我用了win11系统自带的解压工具来解压，然后导致出错了
想来想去我就用winRAR尝试解压文件，然后成功了

**Comment by ToKiNoBug** [comment]
我又上传了一个5.2.2的7z压缩包，现在win11倒是能正确解压7z。
